Pcarioufr/saved view share (#5999)
DataDog/documentation: content/en/logs/explorer/saved_views.md

Select Saved Views directly in the left panel or in the search bar with autocompletion, which matches on either search name or query.

{{< img src="logs/explorer/saved_views/saved_view_load-from-bar.png" alt="Saved Views selection" responsive="true" style="width:90%;" >}}

Star Saved Views to marked them as favorites. Starred Saved Views are available directly from the main navigation menu.

{{< img src="logs/explorer/saved_views/saved_view_load.mp4" alt="Saved Views load" video="true" responsive="true" >}}

### Share a Saved View

> Copy-paste a Saved View short-link to share it with your teammates.

Reviewer: ```suggestion
Copy-paste a Saved View short-link to share it outside of Datadog.
```
